src/sparseml/pytorch/torchvision/export_onnx.py

```diff
@@ -54,13 +56,6 @@
     help="A path to a previous checkpoint to load the state from "
     "and resume the state for exporting, or a zoo stub.",
 )
-@click.option(
```

Comment:
Why are we removing this argument? Don't we want to have both options allowed for the user?